common.go

```diff
@@ -0,0 +1,26 @@
+package hazelcast
```

Comment:
Does it make sense to move the functions in this file to `proxy.go`?

Reply:
Yes that would be better, thanks